gammapy/modeling/models/temporal.py

```diff
@@ -550,7 +558,8 @@ def read(cls, filename, format="table"):
 
         elif format == "map":
             m = RegionNDMap.read(filename)
-            t_ref = time_ref_from_dict(m.meta)
+            header = fits.getheader(filename, "SKYMAP_BANDS")
```

Comment:
Here I would suggest to use `hdulist = fits.open(filename)` and then `RegionNDMap.from_hdulist()`

Reply:
But is this necessary? On `read` the `RegionNDMap.meta` is updated no?